.circleci/config.yml

```diff
@@ -51,6 +51,13 @@ aliases:
           sudo rpm -i -f $(find /var/cache/zypp/packages/ | grep '.rpm$')
         fi
 
+  - &test_junit
+    name: "Enabling JUnit output for Perl tests"
+    command: |
+      yes | sudo cpan TAP::Harness::JUnit
```

Comment:
Why not installing this like any other dependency?

Reply:
It's not packaged

Reply:
Maybe it should be :-)

Reply:
it's at least in `devel:languages:perl`

Reply:
and of course as fresh as the morning breeze!

Reply:
@andrii-suse Can I ask what you would recommend?

- `TAP::Harness::JUnit` is only necessary for CI, it wouldn't make sense to add as a cpam/ rpm dependency
- where should this type of thing be installed? right now it's a separate step in each of the tests
- `devel:languages:perl` is not being used for deps here so far

Btw for docs the questions will be similar as we depend on several asciidoctor packages that aren't normally required.

Reply:
`devel:languages:perl` is a bit dangerous as it normally has more recent versions of packages than `devel:/openQA:/Leap:`.
To me it is a bit similar to Cover dependency - normally we need it only for CI. Thus I think we might add `TAP::Harness::JUnit` to `devel:/openQA:/Leap:/15.1/openSUSE_Leap_15.1`?
Otherwise - I don't think it matters much how we install it, as long as it doesn't interfere with other packages from `devel:/openQA:/Leap:`. It can be explicit step in config.yml `cpan` or `zypper` - I am fine with both. Or it can go to `.circleci/build_cache.sh` after we downloaded all the rest dependencies https://github.com/os-autoinst/openQA/blob/fb95912957c3e34bd915739112f9f47e4313d0c4/.circleci/build_cache.sh#L25

Reply:
@andrii-suse as coolo suggested, submit from devel:languages:perl to openSUSE:Factory and *then* link from Factory into devel:openQA:Leap and all other sub projects

Reply:
once you submitted to Factory, we can also sidetrack this and create a fixed link to dlp there